### DISCUSSION TITLE
How to get the value of the cell when I click it ??

### DISCUSSION THREAD

**Discussion by Gh05tJr** [Akascape/CTkTable#49] [Q&A] [answered]
Can anyone tell how can we get the  value of the cell when I click it ??

**Comment by Akascape** [comment, marked as answer]
@DIY-TECH-HACKER Example given here: https://github.com/Akascape/CTkTable/issues/6#issuecomment-1570224204

**Reply by Gh05tJr** [reply]
Thank you so much for your quick reply.
